src/coreclr/ilasm/portable_pdb.cpp

```diff
@@ -311,8 +325,8 @@ HRESULT PortablePdbWriter::DefineSequencePoints(Method* method)
     // finally define sequence points for the method
     if ((isValid && currSeqPoint != NULL) || hasEmptyMethodBody)
     {
-        mdDocument document = hasEmptyMethodBody ? m_currentDocument->GetToken() : currSeqPoint->pOwnerDocument->GetToken();
-        ULONG documentRid = RidFromToken(document);
+        mdDocument document = hasEmptyMethodBody ? m_currentDocument->GetToken() : method->m_FirstDocument->GetToken();
+        ULONG documentRid = method->m_HasMultipleDocuments ? 0 : RidFromToken(document);
```

Comment:
Because that's what the spec says :)
https://github.com/dotnet/runtime/blob/main/docs/design/specs/PortablePdb-Metadata.md#methoddebuginformation-table-0x31
I think the idea is to not confuse tools that cannot handle multiple documents / not change the semantic from only document to first document.